src/library/BundleUtils/BundleUtils.spec.ts

```diff
@@ -59,7 +59,36 @@ describe("BundleUtils", () => {
         "ec0bb1b3-b229-36cf-7e34-3f5fec9d3afe"
       );
       // validate
-      expect(actual).not.toBeNull();
+      expect(actual).not.toBeUndefined();
```

Comment:
I think to check for undefined makes more sense here because it would only be null if the bundle was empty or nullish